docs/index.md

```diff
@@ -36,6 +36,18 @@ RCS is developed and supported by UAB IT's Research Computing Group. We are deve
 
 RCS is an out growth of the UABgrid pilot, launched in September 2007 which has focused on demonstrating the utility of unlimited analysis, storage, and application for research. RCS is built on the same technology foundations used by major cloud vendors and decades of distributed systems computing research, technology that powered the last ten years of large scale systems serving prominent national and international initiatives like the [Open Science Grid](https://osg-htc.org/), [XSEDE](https://www.xsede.org/), the [LHC Computing Grid](https://wlcg.web.cern.ch/), and [NCIP](https://datascience.cancer.gov/).
 
+## Follow up ServiceNow Tickets
+
+If you are unable to find a reference or correspondence for your already submitted ServcieNow ticket, you can now visit UAB's [ServiceNow portal](https://uabprod.service-now.com/service_portal) to track and follow up your submitted tickets. Visit the portal link, and click the "Log in" button to sign in with your UAB SSO (Single Sign On).
```

Comment:
"follow up" same as above.

I think "Visit the portal link," is redundant here. Try saying "At the website, click the ..."